framework/auth/core.py

```diff
@@ -373,6 +375,30 @@ def create_confirmed(cls, username, password, fullname):
         user.date_confirmed = user.date_registered
         return user
 
+    def get_cookie(self, secret):
```

Comment:
I considered it but thought it was a bit odd to import `website.settings` in `framework`